.misc/deploy.pypi.sh

```diff
@@ -11,11 +11,15 @@ echo "Uploading coala to pypi"
 pip3 install twine wheel
 python3 setup.py sdist bdist_wheel
 # Upload one by one to avoid timeout
-# Don't block merging because pypi is in error 500 mood again
-set +e
-twine upload dist/*.whl -u "$PYPIUSER" -p "$PYPIPW"
-twine upload dist/*.tar.gz -u "$PYPIUSER" -p "$PYPIPW"
-set -e
+twine upload dist/* -u "$PYPIUSER" -p "$PYPIPW" 2>&1 | tee er.txt
```

Comment:
`er.txt` is not very descriptive, and if `er` means `error` it is wrong.  It includes all twine output, not just errors.
Use a descriptive filename.